src/server/main_service.cc

```diff
@@ -871,14 +872,23 @@ std::optional<ErrorReply> Service::VerifyCommandState(const CommandId* cid, CmdA
   return nullopt;
 }
 
+static std::string FullAclCommandFromArgs(CmdArgList args) {
+  ToUpper(&args[1]);
+  // Guranteed SSO no dynamic allocations here
+  return std::string("ACL ") + std::string(args[1].begin(), args[1].end());
+}
+
 void Service::DispatchCommand(CmdArgList args, facade::ConnectionContext* cntx) {
   CHECK(!args.empty());
   DCHECK_NE(0u, shard_set->size()) << "Init was not called";
 
   ServerState& etl = *ServerState::tlocal();
 
   ToUpper(&args[0]);
-  const CommandId* cid = FindCmd(args);
+  const std::string_view command(args[0].begin(), args[0].end());
+  const bool is_acl_command = (command == "ACL");
+
+  const CommandId* cid = is_acl_command ? FindCmd(FullAclCommandFromArgs(args)) : FindCmd(args);
```

Comment:
We need a better mechanism for handling sub-commands, this is a really dirty fix 😵  

This if can be added to FindCmd, we already have there some stream command IIRC fix there